file-browser/src/components/SearchInput.js

```diff
@@ -5,6 +5,6 @@ import React from 'react';
 export default class SearchInput extends React.Component {
     render() {
         const props = this.props;
-        return (<div style={props.style}>SearchInput</div>)
+        return (<div style={props.style}>SearchInputs</div>)
```

Comment:
I think I have to watch some videos to learn React router, this will be done tonight. Tonight also I will go through PropTypes doc, especially those examples in the doc. Tomorrow I have to work at office but I will find some time read luna/luna saga.  Probably cannot finish all the todos by tmr but I will try.